src/Core/AcmeClient.php

```diff
@@ -191,6 +191,27 @@ public function finalizeOrder(CertificateOrder $order, CertificateRequest $csr,
         return $this->createCertificateResponse($csr, Utils::copyToString($response->getBody()));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function requestMultipleAuthorization(array $domains): array 
+    {
+        if (is_array($domains)) {
+            $order = $this->requestOrder($domains);
+
+            $authorizationChallenges = array();
+
+            try {
+                foreach($domains as $domain) {
+                    $authorizationChallenges[] = $order->getAuthorizationChallenges($domain);
+                }
+                return $authorizationChallenges;
```

Comment:
Missing an empty line before the return

Reply:
Just wondering why you didn't merge this change?